pkg/etc/wavefront/wavefront-proxy/wavefront.conf.default

```diff
@@ -3,83 +3,60 @@
 #
 #   Typically in /etc/wavefront/wavefront-proxy/wavefront.conf
 #
-##############################################################################
-# The server should be either the primary Wavefront cloud server, or your custom VPC address.
-#   This will be provided to you by Wavefront.
-#
+########################################################################################################################
+# Wavefront API endpoint URL. This will be provided to you by Wavefront, usually the same as URL of your Wavefront
+# instance, with "/api/" suffix.
```

Comment:
Wavefront API endpoint URL. Usually the same as the URL of your Wavefront instance with a `api` suffix -- or Wavefront provides the URL.